static/merge.js

```diff
@@ -130,51 +132,81 @@
     }
 
     function addEmptyCellBefore(constructId, j, startIndex) {
+        var element = '<td data-csl-id="" class="text-center"></td>';
+
+        if (endClasses.includes(j - 1)) {
+            element = '<td data-csl-id="" class="text-center thicker"' +
```

Comment:
This one is a little messy. Try to use jquery to build this element. It will make it clear what is the difference between those two.